[CLI]: Wandb 0.17.6 breaks lightgbm integration

### Describe the bug

<!--- Description of the issue below  -->
When using Wandb 0.17.6 and lightgbm 4.5.0 we get an attribute error in the lightgbm integration. We do not get the error in Wandb 0.17.5.


```
.venv/lib/python3.10/site-packages/wandb/integration/lightgbm/__init__.py:238: in log_summary
    with wb_telemetry.context() as tel:
.venv/lib/python3.10/site-packages/wandb/sdk/lib/telemetry.py:42: in __exit__
    self._run._telemetry_callback(self._obj)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <wandb.sdk.wandb_run.Run object at 0x72f5c4583940>, telem_obj = feature {
  lightgbm_log_summary: true
}


    def _telemetry_callback(self, telem_obj: telemetry.TelemetryRecord) -> None:
>       self._telemetry_obj.MergeFrom(telem_obj)
E       AttributeError: 'Run' object has no attribute '_telemetry_obj'. Did you mean: '_telemetry_flush'?

.venv/lib/python3.10/site-packages/wandb/sdk/wandb_run.py:799: AttributeError
```


### Additional Files

_No response_

### Environment

WandB version: 0.17.6

OS: Ubuntu 

Python version: Python 3.10.12

Versions of relevant libraries:


### Additional Context

_No response_

--- a/wandb/sdk/wandb_run.py
+++ b/wandb/sdk/wandb_run.py
@@ -796,11 +796,15 @@ class Run:
                     self._unique_launch_artifact_sequence_names[sequence_name] = item
 
     def _telemetry_callback(self, telem_obj: telemetry.TelemetryRecord) -> None:
+        if not hasattr(self, "_telemetry_obj"):
+            return
         self._telemetry_obj.MergeFrom(telem_obj)
         self._telemetry_obj_dirty = True
         self._telemetry_flush()
 
     def _telemetry_flush(self) -> None:
+        if not hasattr(self, "_telemetry_obj"):
+            return
         if not self._telemetry_obj_active:
             return
         if not self._telemetry_obj_dirty:

